CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ cmake_minimum_required(VERSION 3.15)
 project(sdklauncher CXX)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
+# Qt
```

Comment:
Any reason why you essentially redid the structure for the cmake file?

Reply:
I believe I copied it from an internal branch of the SDK Launcher. it doesn't change much, can be reverted if needed.

Reply:
It's fine, was just a bit curious.